exercise-book/src/nrf52-usb-supporting-standard-requests.md

```diff
@@ -1,21 +1,171 @@
 # USB-4: Supporting more Standard Requests
 
-After responding to the `GET_DESCRIPTOR Device` request the host will start sending different requests.
+After responding to the `GET_DESCRIPTOR Device` request the host will start sending different requests. Let's identify those, and then handle them.
 
-✅ Update the parser in `nrf52-code/usb-lib` so that it can handle the following requests:
+## Update the parser
 
-1. `GET_DESCRIPTOR Configuration`, see the section on [Handling GET_DESCRIPTOR Configuration Requests](./nrf52-usb-get-descriptor-config.md#handling-get_descriptor-configuration-requests)
-2. `SET_CONFIGURATION`, see the section on [SET_CONFIGURATION](./nrf52-usb-getting-device-configured.md#set_configuration) of this course material
+The starter [`nrf52-code/usb-lib`](../../nrf52-code/usb-lib) package contains unit tests for everything we need. Some of them have been commented out using a `#[cfg(TODO)]` attribute - you can go ahead and remove all of those attributes now so that everything is enabled. Now fix those new tests!
```

Comment:
> you can go ahead and remove all of those attributes no so that everything is enabled. Now fix those new tests!

This is phrased as step that people should be doing. I think for consistency it should have a ✅ and start on a newline.
```suggestion
The starter [`nrf52-code/usb-lib`](../../nrf52-code/usb-lib) package contains unit tests for everything we need. Some of them have been commented out using a `#[cfg(TODO)]` attribute.

 ✅ Remove all `#[cfg(TODO)]` attributes so that everything is enabled. Fix those new tests!
```

Reply:
"Fix those new tests!" is covered by the two points below, but yes, it should have a tick.